server/sonar-web/src/main/js/components/common/BranchStatus.tsx

```diff
@@ -52,21 +55,37 @@ export default function BranchStatus({ branch, concise = false }: Props) {
       </ul>
     ) : (
       <ul className="branch-status">
-        <li className="spacer-right">
+        <li className="little-spacer-right">
           <StatusIndicator color={indicatorColor} size="small" />
         </li>
         <li className="spacer-left">
           {branch.status.bugs}
-          <BugIcon />
+          <BugIcon className="little-spacer-left" />
         </li>
         <li className="spacer-left">
           {branch.status.vulnerabilities}
-          <VulnerabilityIcon />
+          <VulnerabilityIcon className="little-spacer-left" />
         </li>
         <li className="spacer-left">
           {branch.status.codeSmells}
-          <CodeSmellIcon />
+          <CodeSmellIcon className="little-spacer-left" />
         </li>
+        {shouldDisplayHelper && (
+          <li className="spacer-left">
+            <Tooltip
+              overlay={translateWithParameters(
+                'branches.short_lived.quality_gate.description',
+                totalIssues <= 1
+                  ? translateWithParameters('x_issue', totalIssues)
+                  : translateWithParameters('x_issues', totalIssues)
+              )}
+              placement="right">
+              <span>
```

Comment:
I guess you can put `<Tooltip />` around `<li />` to avoid this redundant `<span />`